File: eTeatar/eTeatar.Model/Predstava.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eTeatar.Model
{
    public class Predstava
    {
        public int PredstavaId { get; set; }

        public string Naziv { get; set; }

        public string Opis { get; set; }

        public int Trajanje { get; set; }

        public string Produkcija { get; set; }

        public string Koreografija { get; set; }

        public string Scenografija { get; set; }

        public decimal Cijena { get; set; }

        public byte[]? Slika { get; set; }
    }
}

File: eTeatar/UI/eteatar_mobile/lib/models/predstava.dart
import 'package:json_annotation/json_annotation.dart';

part 'predstava.g.dart';

@JsonSerializable()
class Predstava {
  int? predstavaId;
  String? naziv;
  double? cijena;
  String? slika;
  String? opis;
  String? produkcija;
  String? koreografija;
  String? scenografija;
  List<int>? zanrovi; 
  List<int>? glumci; 
  int? trajanje;

  Predstava(
    this.predstavaId,
    this.naziv
  );
  factory Predstava.fromJson(Map<String, dynamic> json) => _$PredstavaFromJson(json);

  /// Connect the generated [_$PredstavaToJson] function to the `toJson` method.
  Map<String, dynamic> toJson() => _$PredstavaToJson(this);
}